specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/examples/WhoIsBlueprint_Action.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "scope": "subscriptions/f8df94f2-2f5a-4f4a-bcaf-1bb992fb564b",
```

Comment:
>f8df94f2-2f5a-4f4a-bcaf-1bb992fb564b [](start = 28, length = 36)

You should probably use a placeholder here like {subscriptionId}. This will avoid disclosing an existing subscription publically.

Reply:
Done